### DISCUSSION TITLE
Please consider crypto payments for your awesome software

### DISCUSSION THREAD

**Discussion by lanmaster** [d4rken-org/sdmaid-se#1377] [Q&A] [answered]
Users from some countries cannot be a sponsor on some politic cases. Please consider BTC or some other...

**Comment by d4rken** [comment, marked as answer]
If you are unable to buy on Google Play or use one of the existing Sponsor methods, then you are free to use the FOSS build without making a donation. The check in the FOSS version is based on the honor system.

I'm not against crypto, but too few people would use it, to make it worth the overhead. It would not just be posting a wallet address, I'd have setup accounting and tax handling for it too.

**Reply by lanmaster** [reply]
You are the best!
